### ISSUE TITLE
Oswald doesn't render font weight correctly when printing a webpage

### ISSUE THREAD

**Issue by Nazrinn** [google/fonts#2734]
Hi! 
I am currently using Oswald on a website meant to be printed (it's my resume). However, I have noticed Oswald doesn't render the font weight it was assigned to in CSS no matter what. This is a problem I have even found on the Google Fonts website, as demonstrated here.

 Normal web page view:
![image](https://user-images.githubusercontent.com/36900693/95551843-8fcc0d80-09d9-11eb-846f-c8c55a2abf54.png)

When trying to print the page:
![image](https://user-images.githubusercontent.com/36900693/95551950-c7d35080-09d9-11eb-9e48-71251c9a7c04.png)

Has anybody any idea of why that is and on how to fix it?

Thanks!

**Comment by RosaWagner** [comment]
Hi, thanks for reporting this. 

I can’t replicate the bug on my machine (Mac OS 10.14.6) whatever the browser I use.

Can you give more details about:
- the browser you are using + version
- the OS you are on + version
- About the second picture: is it a screenshot of the printing preview? What is the result when printed ?

**Comment by Nazrinn** [comment]
Browser: 
`Google Chrome - version 85.0.4183.121`

OS: 
`Windows 10 - version 1909 - Build 18363.1082`

The screenshot is from the printing preview only. This is a problem I have spotted both in Chrome and Edge. In Firefox and Opera, I do not have this problem.

Unfortunately my printer is not cooperating at the moment, but if I save the document in PDF, I still get the weirdly bold font. See here:
[Oswald-GoogleFonts(Chrome).pdf](https://github.com/google/fonts/files/5353685/Oswald-GoogleFonts.Chrome.pdf)
[Oswald-GoogleFonts(Firefox).pdf](https://github.com/google/fonts/files/5353688/Oswald-GoogleFonts.Firefox.pdf)


I'd like to also add that if I use the font locally in ttf, I do not seem to have that problem.

**Comment by Nazrinn** [comment]
Update: the printer cooperated. The printer does seem to print out the weirdly bold font as well (printed from Chrome).

![20201009_050136](https://user-images.githubusercontent.com/36900693/95564312-9dd75980-09ec-11eb-921b-358613792882.jpg)

**Comment by RosaWagner** [comment]
Thanks. Can you try with another variable font, a recent one, like Piazzolla ?

**Comment by thlinard** [comment]
On macOS 10.15.7, I can only reproduce the problem with Chrome:

- https://github.com/google/fonts/files/5353736/Oswald.-.Google.Fonts.-.Safari.14.0.pdf
- https://github.com/google/fonts/files/5353737/Oswald.-.Google.Fonts.-.Firefox.81.0.1.pdf
- https://github.com/google/fonts/files/5353738/Oswald.-.Google.Fonts.-.Chrome.86.0.4240.75.pdf

**Comment by Nazrinn** [comment]
Looks like that's it. It doesn't render correctly either.

[Piazzolla-GoogleFonts(Chrome).pdf](https://github.com/google/fonts/files/5353740/Piazzolla-GoogleFonts.Chrome.pdf)

Same in Edge. Perhaps this is a Chromium problem?

**Comment by thlinard** [comment]
This is weird. I cannot obtain twice the same result with Chrome. Here another attempt (correct!):
https://github.com/google/fonts/files/5353800/Oswald.-.Google.Fonts.-.Chrome.86.0.4240.75.-.Test.2.pdf

**Comment by RosaWagner** [comment]
Merci Thomas. No problem on my side on Mojav, same Chrome, skia/PDF 86m.
[Oswald - Google Fonts.pdf](https://github.com/google/fonts/files/5353885/Oswald.-.Google.Fonts.pdf)

**Comment by thlinard** [comment]
Same Chrome? But you have "Skia/PDF m86"! So Chrome has two Skia versions?? Definitively very weird…

**Comment by RosaWagner** [comment]
@Nazrinn, if your main goal is to be able to print from the browser, probably safer for now to use self-hosted static TTF. If you wanted a custom instance you can export it with axis-praxis/samsa, or using fonttools.

**Comment by Nazrinn** [comment]
Ah. And I thought I could have it easy by just exporting Google Fonts. I haven't heard of the two other tools. I think I'll stick to local fonts for now for the sake of compatibility since I assume a fix won't come out globally any time soon. Thanks for the tips tough @RosaWagner!

**Comment by thlinard** [comment]
@RosaWagner: you're right, I corrected my comments. Definitively a Windows problem, not a Mac one.
